src/gsi/setupdbz.f90

```diff
@@ -1779,6 +1781,7 @@ subroutine init_netcdf_diag_
      end if
 
      call nc_diag_init(diag_conv_file, append=append_diag)
+     call flush(6)
```

Comment:
Those "call flush". lines remaining from previous coding/debugging process could be removed